interface/globals.php

```diff
@@ -287,6 +287,14 @@ function undoMagicQuotes($array, $topLevel=true) {
       if ($gl_value == '2') $GLOBALS['sell_non_drug_products'] = 1;
       else if ($gl_value == '3') $GLOBALS['sell_non_drug_products'] = 2;
     }
+    else if ($gl_name == 'gbl_time_zone') {
+      // If value is empty default to old behavior.
+      if ($gl_value) {
+        date_default_timezone_set($gl_value);
+        // Synchronize MySQL time zone with PHP time zone.
+        sqlStatement("SET time_zone = ?", array((new DateTime())->format("P")));
```

Comment:
OK I see:

    common/database/connector.php
    gacl/gacl.class.php
    interface/main/calendar/includes/pnAPI.php
    interface/modules/zend_modules/config/autoload/global.php
    library/classes/Installer.class.php
    portal/patient/_machine_config.php

Do any of these care what the MySQL time zone is?  Typically that matters if you use something like NOW() in a query.